gsoc/proposal-Data-Visualization-For-Archived-Content.md

```diff
@@ -0,0 +1,97 @@
+
+**Milestones/Timeline:**  
+This project is essentially composed of **3 major milestones**: working with my mentors to identify which data is meaningful and worth developing a model for, developing the model while simultaneously receiving feedback on improvements, and then eventually committing it to production for use. Below you will find a rough timeline:
+
+* **Before May 30th**
+Familiarize myself with the current codebase of the app, any relevant API’s that DataRefuge provides, as well as, set up my environment to run local versions of your tools/apps. I will also work with my mentor to make sure that I have full understanding of the codebase as well as overall goals.
+
+* **May 30th - June 13th**
+I will spend this time working on communicating with my mentor in order to understand the current toolkit in how it captures data as well as trying to identify which of the data is relevant from both EDGI and DataRefuge. 
+
+* **Jun 13th - Jun 27th**
+Once I have identified relevant data, I will start working on potential design mockups for how this information could be displayed as well as integrated with the React.JS app . I will also be continuously communicating with any relevant teams to gain feedback on my designs and improve them.
+
+* **Jun 27th - Jul 11th**
+Once I have finalized a set of designs for graphs that both the developers and I agree on as being the most “impactful” to individuals, I will start working on prototypes for these designs. I will also communicate directly with the team responsible for developing tools that archive this data so that they can potentially develop a set of API calls that I might be able to make directly to the tool instead of pulling it from a 3rd party.
+
+* **Jul 11th -  Jul 25th**
+I will have mostly completed any models at this time and will submit it to my mentors for review. If any changes and/or any new models need to be made, I will work on them as well! Most of my time this period will be spent communicating with the team and listening to any feedback they have.
+
+* **Jul 25th - Aug 8th**
+Finalize any ideas I had and hopefully commit my developed models to the “live” website 
+
+* **Aug 8th - Aug 22nd**
+Reach out to individuals using the website as well as my mentors to see if I can identify any other changes to further optimize the experience and make it more interactive and meaningful on the end-user. As with any product being shipped, testing usage and interaction with the product is important to gauge its overall perception. 

```

Comment:
I also appreciate the explanations of what is expected during each time period. All too often, proposals have only a bullet list and leaves too much to the imagination.